opm/parser/eclipse/EclipseState/Schedule/WellInjectionProperties.cpp

```diff
@@ -54,4 +55,20 @@ namespace Opm {
     bool WellInjectionProperties::operator!=(const WellInjectionProperties& other) const {
         return !(*this == other);
     }
+
+    std::ostream& operator<<( std::ostream& stream,
```

Comment:
Is  `operator<<` really required? When / if we decide to support output from the parser state I would like a slightly coherent discussion on the topic, so that we agree on one way to do it.